src/Compilers/CSharp/Portable/Parser/SlidingTextWindow.cs

```diff
@@ -320,8 +320,9 @@ public char PeekChar(int delta)
             this.AdvanceChar(delta);
 
             char ch;
-            if (_offset >= _characterWindowCount
+            if ((_offset >= _characterWindowCount
                 && !MoreChars())
+                || _offset < 0)
```

Comment:
>_offset [](start = 19, length = 7)

When is `_offset` less than 0?

Reply:
This should be removed.

---
In reply to: [466643625](https://github.com/dotnet/roslyn/pull/46520#discussion_r466643625) [](ancestors = 466643625)

Reply:
It has been in commit 19.